lib/gds_api/govuk_headers.rb

```diff
@@ -0,0 +1,14 @@
+module GdsApi
+  class GovukHeaders
+    class << self
+      def headers
+        Thread.current[:headers] ||= {}
+      end
+
+      def values
+        headers.select {|k, v| !(v.nil? || v.empty?) }
```

Comment:
I think `present?` is a Rails-specific method, the adapters need to work for non-Rails apps too.